documentation/configuration/configfile.md

```diff
@@ -98,6 +98,10 @@ Here is a sample configuration file:
 # by connectRetries. (default: 0)
 # flyway.connectRetries=
 
+# The maximum time between retries when attempting to connect to the database in seconds.
+# This will cap the interval between connect retry to the value provided. (default: 120)
+# flyway.connectRetriesInterval=
+
```

Comment:
We don't need to manually modify this file, it gets auto updated based on the sample config file in the repo